locales/uk/json.json

```
@@ -460,7 +460,7 @@
     "Loading Notifications": "Завантаження повідомлень",
     "Log in": "Авторизувати",
     "Log In": "Увійти",
-    "Log Out": "вийти з системи",
+    "Log Out": "Вийти з системи",
     "Log out": "Вийти",
```

```suggestion
    "Log out": "Вийти з системи",
```
Як Ви думаєте, може, тут теж перейменувати ключ, щоб відповідати перекладу інших?

---

Мені просто не давала спокою маленька літера в цій фразі 😅 Вона використовується в меню і відразу помітна. Але пізніше я планую переглянути всі переклади і запропонувати більше змін

---

Розумію) Це був один із доданих ключів і на ньому спрацював автоперекладач.